sdk/eventhub/event-hubs/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## 5.2.1 (Unreleased)
 
+- Updates the `peer.address` attribute on "Azure.EventHubs.send" spans to refer
+  to the fully qualified namespace instead of the endpoint.
+  Fixes [#7109](https://github.com/Azure/azure-sdk-for-js/issues/7109)
```

Comment:
If we are going to call out any tracing related changes in the changelog, I'd like to add a warning that tracing as a feature is still in preview and that the apis can have breaking changes in future minor/patch updates.

Reply:
I added a note. Since I plan to have one more tracing update might make sense to group them together with one note. I can change that in the next PR.